cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ArtifactHttpHandler.java

```diff
@@ -515,24 +518,35 @@ public void getArtifactPlugin(HttpRequest request, HttpResponder responder,
                                 @PathParam("artifact-version") String artifactVersion,
                                 @PathParam("plugin-type") String pluginType,
                                 @PathParam("plugin-name") String pluginName,
-                                @QueryParam("scope") @DefaultValue("user") String scope,
-                                @QueryParam("versionRange") String versionRange,
+                                @QueryParam("scope") @DefaultValue("user") final String scope,
+                                @QueryParam("pluginArtifactName") final String pluginArtifactName,
+                                @QueryParam("pluginVersion") String pluginVersion,
```

Comment:
should be consistent with the naming, let's use 'artifactName', 'artifactVersion', 'artifactScope' for the query params (can use pluginArtifactName, pluginArtifactVersion, etc. for the variable names).